src/lib.rs

```diff
@@ -1172,6 +1172,10 @@ pub unsafe extern "C" fn wgpuCommandEncoderBeginRenderPass(
             make_slice(descriptor.colorAttachments, descriptor.colorAttachmentCount)
                 .iter()
                 .map(|color_attachment| {
+                    if color_attachment.depthSlice != 0 {
```

Comment:
```suggestion
if color_attachment.depthSlice != native::WGPU_DEPTH_SLICE_UNDEFINED {
                        log::warn!("Depth slice on color attachments is not implemented");
}
```
and then in examples update accordingly:

```c
(const WGPURenderPassColorAttachment){
    ...
    .depthSlice = WGPU_DEPTH_SLICE_UNDEFINED,
```